python/cdp/test/test_cdp_client_interface.py

```diff
@@ -39,6 +39,10 @@ def is_method_wrapped(api_method, evm_methods, solana_methods):
         stripped_method = api_method.replace("solana_", "").replace("_by_name", "")
         if stripped_method in solana_methods:
             return True
+    elif "sol" in api_method:
```

Comment:
we now have methods like `update_sol_account` so need this change.
we can push back on the backend so that all solana functions use the entire word i.e. `update_solana_account`, but i figured we could be flexible here.

Reply:
Got it – @sammccord @jazz-cb, should the API spec have used the term `solana` here instead of `sol`? Is this a change we want to make, or leave as-is?